src/stores/useHoloStore.js

```diff
@@ -19,7 +20,9 @@ const makeUseHoloStore = ({ connectionArgs, MockWebSdk }) => defineStore('holo',
     // These two values are subscribed to by clientStore
     isReady: false,
     appInfo: null,
-    kycLevel: null
+    kycLevel: null,
+    dashboardStatistics: emptyHappStatistics,
+    allHappStatistics: []
```

Comment:
this stuff properly belongs in it's own store in cloud console. something like `happStatsStore`